src/dataflow/src/logging/timely.rs

```diff
@@ -22,8 +22,9 @@ use timely::logging::{ParkEvent, TimelyEvent, WorkerIdentifier};
 
 use super::{LogVariant, TimelyLog};
 use crate::activator::RcActivator;
-use crate::arrangement::manager::RowSpine;
-use crate::arrangement::KeysValsHandle;
+use mz_dataflow_types::KeysValsHandle;
+use mz_dataflow_types::RowSpine;
+
```

Comment:
The way these 2 sections are laid out is a bit weird now (as well as the other changes like this one), overall to match the rest of the repo it should probably be

```
use std::whatever;
use std::whatever2;

use external_crate::a;
use external_crate2::b;

use mz_crate::stuff;
use mz_crate2::b;

use super::stuff;
use crate::crate_stuff;
```

Reply:
unfortunately rustfmt only sorts within each block separated by spaces, and the options that make it split imports into sections are nightly :(

Reply:
I just type `cargo fmt`. Happy to have folks with additional style requirements come in afterwards!